src/main/java/uk/gov/pay/products/validations/PANDetector.java

```diff
@@ -0,0 +1,19 @@
+package uk.gov.pay.products.validations;
+
+import org.apache.commons.validator.routines.checkdigit.LuhnCheckDigit;
+
+public class PANDetector {
+    private static final Integer MIN_PAN_LENGTH = 11;
+    private static final Integer MAX_PAN_LENGTH = 19;
```

Comment:
These can be primitive `int`s. It looks like we use a minimum of 12 not 11 in frontend: https://github.com/alphagov/pay-frontend/blob/437a67cb8c50a9368bb767d9c6b2ca5ce4dee784/app/utils/charge-validation-fields.js#L161